NitroxClient/Communication/DeferringPacketReceiver.cs

```diff
@@ -49,42 +49,24 @@ public Queue<Packet> GetReceivedPackets()
 
         private bool PacketWasDeferred(Packet packet)
         {
-            if (packet is PlayerActionPacket)
+            if (packet is RangedPacket)
             {
-                PlayerActionPacket playerAction = (PlayerActionPacket)packet;
+                RangedPacket playerAction = (RangedPacket)packet;
 
-                if (!playerAction.PlayerMustBeInRangeToReceive)
+                if (visibleCells.Contains(playerAction.AbsoluteEntityCell))
                 {
                     return false;
                 }
 
-                AbsoluteEntityCell cell = new AbsoluteEntityCell(playerAction.ActionPosition);
-
-                bool cellLoaded = false;
-
-                for (int level = 0; level <= DESIRED_CELL_MIN_LOD_FOR_ACTIONS; level++)
-                {
-                    VisibleCell visibleCell = new VisibleCell(cell, level);
-
-                    if (visibleCells.HasVisibleCell(visibleCell))
-                    {
-                        cellLoaded = true;
-                        break;
-                    }
-                }
-
-                if (!cellLoaded)
-                {
-                    Log.Debug("Action was deferred, cell not loaded (with required lod): " + cell);
-                    AddPacketToDeferredMap(playerAction, cell);
-                    return true;
-                }
+                Log.Debug($"Action {packet} was deferred, cell not loaded (with required lod): {playerAction.AbsoluteEntityCell}");
+                AddPacketToDeferredMap(playerAction, playerAction.AbsoluteEntityCell);
+                return true;
             }
 
             return false;
         }
 
-        private void AddPacketToDeferredMap(PlayerActionPacket playerAction, AbsoluteEntityCell cell)
+        private void AddPacketToDeferredMap(RangedPacket playerAction, AbsoluteEntityCell cell)
```

Comment:
minor: change the variable naming

Reply:
I wasn't entirely sure about the `Ranged` naming in the first place, there should be a better synonym. Completely forgot to add that question to the PR, so if you have a better name please let me know; I'll add it to #172.

Reply:
Yeah ranged doesn't really tell you its use - unless we put a boat load of class documentation.  We could say DeferrablePacket.

Reply:
That sounds like a proper alternative.